addons/csw/stringtable.xml

```diff
@@ -404,6 +404,7 @@
         </Key>
         <Key ID="STR_ACE_CSW_StaticM2ShieldBag_displayName">
             <English>[CSW] Static M2 w/ Shield</English>
+            <German>[CSW] Statische M2 mit Schild</German>
```

Comment:
Neutrum

Reply:
'Statische' ist dabei aber einheitlich mit den anderen Einträgen, z. B.: '<German>[CSW] Statische XM312</German>'

Reply:
Für mich heißt es "das M2" und nicht "die M2". Wenn es in anderen ACE-Übersetzungen falsch ist, dann ist es dort auch falsch. Wenn es in Vanilla-Übersetzungen falsch ist, dann kann man es konsistent so falsch lassen, denke ich.

Reply:
Habe mal in der Übersetzung von Arma 3 nachgeschaut:
![image](https://user-images.githubusercontent.com/33136944/102806096-8b01dd80-43bc-11eb-87ae-c0e8c22d38a9.png)

Reply:
"Der Werfer", "das M2".